site/webpack.config.js

```diff
@@ -0,0 +1,68 @@
+var Encore = require('@symfony/webpack-encore');
+
+Encore
+// directory where compiled assets will be stored
+    .setOutputPath('web/build/')
+    // public path used by the web server to access the output path
+    .setPublicPath('/build')
+    // only needed for CDN's or sub-directory deploy
+    //.setManifestKeyPrefix('build/')
+
+    /*
+     * ENTRY CONFIG
+     *
+     * Add 1 entry for each "page" of your app
+     * (including one that's included on every page - e.g. "app")
+     *
+     * Each entry will result in one JavaScript file (e.g. app.js)
+     * and one CSS file (e.g. app.css) if you JavaScript imports CSS.
+     */
+    //css entries
+    .addStyleEntry('app-css', ['./web/assets/scss/librecores.scss',
+        './web/assets/scss/bootstrap-librecores.scss',
+        './web/assets/css/bootstrap-social.css',
+        './web/assets/css/font-awesome.min.css',
+        './web/assets/css/librecores.css'])
+    .addStyleEntry('trumbowyg', ['./web/assets/scss/trumbowyg.scss'])
+    .addStyleEntry('project_new_css','./web/assets/css/bootstrap-select.css')
```

Comment:
* space after the `,`
* for consistency, always give the second argument as array